app/src/main/java/at/bitfire/icsdroid/model/EditSubscriptionModel.kt

```diff
@@ -20,7 +20,7 @@ class EditSubscriptionModel(
     private val credentialsDao = db.credentialsDao()
     private val subscriptionsDao = db.subscriptionsDao()
 
-    val successMessage = MutableLiveData<String>()
+    val successMessage = mutableStateOf<String?>(null)
```

Comment:
Again, `var successMessage by mutableStateOf<String?>(null)` (+`private set`)

Reply:
will create another data class for consistency